### PR TITLE
2479 buttons in steps

### PR DESCRIPTION
**Pull request by easherma**
hey this works alright!
Didn't require any backend changes, just some judicious hacking and reading of the html-parser docs
Fixes https://github.com/cityofaustin/techstack/issues/2479 , in terms of stepsWithOptions at least

### CODE REVIEW

File: src/components/Steps/StepWithOptions.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Parser from 'html-react-parser';
+import Parser, { domToReact } from 'html-react-parser';
```
Comment (briaguya) [inline src/components/Steps/StepWithOptions.js]: Is there a reason we need to import `domToReact` here? Doesn't seem to be something we're using

Comment (easherma) [reply src/components/Steps/StepWithOptions.js]: Ah no I was trying previously but forgot to remove

**Comment by easherma** [comment]
Next steps:
Figure out what publishing the service page might look like
Probably remove/modify some routing somewhere to forms.austin.gov on the OPO department page
Test with more than one service page, see if the fix is needed for other kinds of steps

**Comment by briaguya** [comment]
Screen Shot 2019-07-17 at 11 30 35 PM
Screen Shot 2019-07-17 at 11 30 42 PM

I'm pretty sure we'll need to be able to support both buttons and standard link style links in there. @chasechenevert any design input?

**Comment by toribr** [comment]
@briaguya Yes, you are correct. We don't want all the links converted to buttons. In fact, the only button that should be converted is the "Start" button in the first option.

**Comment by easherma** [comment]
Lets close this, we went with a different route

**Pull request closed without merging**